kroxylicious-kms-provider-aws-kms/DEV_GUIDE.md

```diff
@@ -29,11 +29,11 @@ your Kafka client off-EC2.
    ```
 8. Scp a Kroxylicious dist to the EC2 instance.
    ```bash
-   scp  -i "~/.ssh/MyEC2KeyPair.pem" /kroxylicious-app-0.9.0-SNAPSHOT-bin.zip ec2-user@ec2-xxx-xxx-xxx-xxx.compute-1.amazonaws.com:.
+   scp  -i "~/.ssh/MyEC2KeyPair.pem" /kroxylicious-app-0.10.0-SNAPSHOT-bin.zip ec2-user@ec2-xxx-xxx-xxx-xxx.compute-1.amazonaws.com:.
```

Comment:
maybe these usages should be `*` globs or use `x.y.z` to make it clearer it's just an example version

Reply:
yeah, a glob is right approach here.

Reply:
updated